sdk/storage/azure-storage-file-share/azure/storage/fileshare/_shared_access_signature.py

```diff
@@ -14,62 +14,63 @@
 from ._shared import sign_string
 from ._shared.constants import X_MS_VERSION
 from ._shared.models import Services
-from ._shared.shared_access_signature import SharedAccessSignature, _SharedAccessHelper, QueryStringConstants
 from ._shared.parser import _str
```

Comment:
This is sort of unrelated to your change but can you also remove the usage of this entirely from this file? Just use `str` directly in place of it. This is a Python 2 compat thing if you look at the definition of `_str`.